indico/legacy/pdfinterface/latex.py

```diff
@@ -457,7 +457,6 @@ def __init__(self, event, user, contribs=None, tz=None, sort_by=""):
             'event': event,
             'tz': timezone(tz or event.timezone),
             'url': event.url,
-            'fields': [f for f in event.contribution_fields if f.is_active],
```

Comment:
I guess this was unused?

Reply:
I will double check, but seems like unusued